content/sensu-go/6.2/observability-pipeline/observe-schedule/agent.md

```diff
@@ -771,6 +771,7 @@ Usage:
   sensu-agent start [flags]
 
 Flags:
+      --agent-managed-entity                  manage this entity via the agent
```

Comment:
I do not see this flag listed when I run `sensu-agent start --help`

Reply:
Make sure to pull the latest changes and rebuild your agent:

```
$ sensu-agent -h
start the sensu agent

Usage:
  sensu-agent start [flags]

Flags:
      --agent-managed-entity                manage this entity via the agent
[...]
```

Reply:
Also, it's probably only available on sensu-go at the moment, not sensu-enterprise-go yet.